### ISSUE TITLE
Duplicate Key erros when running Workload D with more than 1k records

### ISSUE THREAD

**Issue by andrehgdias** [brianfrankcooper/YCSB#1472]
I'm having duplicate key errors and I'm following the workload sequence suggested at the wiki.

**Comment by Chrisdrinkingcaffee** [comment]
Me too. And WorkloadE also does.

**Comment by andrehgdias** [comment]
@Chrisdrinkingcaffee true, they both are giving that error, despite we delete the database and reload it using the suggested workloads.

**Comment by andrehgdias** [comment]
Does someone have any news about this issue?

**Comment by allanbank** [comment]
Can you verify that the 'load' and "run' workload configurations have the same recordcount?

**Comment by andrehgdias** [comment]
> Can you verify that the 'load' and "run' workload configurations have the same recordcount?

Yes it does. They both have the same configuration.
